python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -95,17 +95,18 @@ Once the smallest size :math:`\sampleSize`  has been estimated, a sample of size
 :math:`\sampleSize` can be
 generated from
 :math:`X` and an upper bound of :math:`x_{\alpha}` is estimated using
-:math:`x_{(\sampleSize-i)}` i.e. the :math:`\sampleSize - i`-th observation
+:math:`x_{(\sampleSize-k)}` i.e. the :math:`\sampleSize - k`-th observation
 in the ordered sample :math:`(x_{(1)}, \dots, x_{(\sampleSize)})`.
 
 
 .. topic:: API:
 
-    - See :class:`~openturns.Wilks`
+    - See :class:`~openturns.experimental.QuantileConfidence`
 
 .. topic:: Examples:
 
     - See :doc:`/auto_data_analysis/manage_data_and_samples/plot_quantile_estimation_wilks`
+    - See :doc:`/auto_data_analysis/manage_data_and_samples/plot_quantile_estimation_chemical_process`
```

Comment:
As this is the theoretical doc, I think we should add the detail of the solution and gather the results in a table, depending on the value of $\alpha, \beta$, as it is written in the Cahier des charges, no?

Furthermore, why don't we detail the case A: we seek the greatest k such that 
$P(X_{(k)} \leq x_\alpha) \geq \beta$ ?

and the case C?